test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/InputTagHelperTest.cs

```diff
@@ -108,10 +108,9 @@ public static TheoryData MultiAttributeCheckBoxData
             await tagHelper.ProcessAsync(context, output);
 
             // Assert
-            Assert.Empty(output.Attributes); // Moved to Content and cleared
-            Assert.Equal(expectedContent, HtmlContentUtilities.HtmlContentToString(output.Content));
-            Assert.Equal(TagMode.SelfClosing, output.TagMode);
-            Assert.Null(output.TagName); // Cleared
+            Assert.NotNull(output.PostElement);
+            Assert.Equal(originalContent, HtmlContentUtilities.HtmlContentToString(output.Content));
+            Assert.Equal(expectedContent, HtmlContentUtilities.HtmlContentToString(output));
```

Comment:
Will likely be cleaner to change the other updated tests similarly. But, I'm not going to hold you to that.